server/accesslog_test.go

```diff
@@ -120,6 +121,7 @@ func TestHandleHTTPAccessLogs(t *testing.T) {
 			orgName:      extAuthzFields[headerOrganization].GetStringValue(),
 			envName:      extAuthzFields[headerEnvironment].GetStringValue(),
 			analyticsMan: testAnalyticsMan,
+			ready:        util.NewAtomicBool(true),
```

Comment:
Why do we need this field here if the access log is always listening? Same Q for the other line in this file.

Reply:
Yeah, we don't. Just wasn't paying close attention to it. :)